turbine/src/broadcast_stage.rs

```diff
@@ -459,14 +472,14 @@ pub fn broadcast_shreds(
     }
     send_mmsg_time.stop();
     transmit_stats.send_mmsg_elapsed += send_mmsg_time.as_us();
-    for (shred, addr) in &quic_packets {
-        let conn = quic_connection_cache.get_connection(addr);
-        if let Err(err) = conn.send_data(shred) {
+    transmit_stats.total_packets += packets.len() + quic_packets.len();
+    for (shred, addr) in quic_packets {
+        let shred = Bytes::from(shred.clone());
+        if let Err(err) = quic_endpoint_sender.blocking_send((addr, shred)) {
```

Comment:
I've always wondered, would there be a big benefit to moving all network stuff to async? My sense has always been "no" since there's such a huge volume of data going around

Reply:
hmm, that is a good point. There might be some benefits in moving more stuff over to tokio runtime. I do not know how much overhead it adds but it definitely can make the code more ergonomic and experessive.

> there's such a huge volume of data going around

Why that should matter either way?

Reply:
I mean that most likely the blocking isn't holding back very much, since all of the data fits neatly into udp packets and has strict timeouts all over the place. I've found that async helps a ton with large amounts of data or longer blocking times, but I would love to do some benchmarks to see if an async runtime helps the validator case.